frontend/src/Room.vue

```diff
@@ -19,6 +19,7 @@
         <svg-icon v-else type="mdi" :path="mdiMicrophone"></svg-icon>
       </button>
     </div>
+    <audio id="audio" ref="audio" hidden autoplay></audio>
```

Comment:
There is no hidden attribute on the audio element

Reply:
removed 'hidden'